runtime/caml/mlvalues.h

```diff
@@ -62,11 +54,16 @@ extern "C" {
          This is for use only by the GC.
 */
 
+/* Needed here for domain_state */
```

Comment:
This comment is no longer relevant, I think.

Reply:
less is more, fixed this.